app/Makefile

```diff
@@ -258,7 +258,7 @@ ingest-bem-pdfs: check-ingest-arguments
 	$(PY_RUN_CMD) ingest-bem-pdfs "$(DATASET_ID)" "$(BENEFIT_PROGRAM)" "$(BENEFIT_REGION)" "$(FILEPATH)"
 
 ingest-edd-web: check-ingest-arguments
-	$(PY_RUN_CMD) ingest-edd-web "$(DATASET_ID)" "$(BENEFIT_PROGRAM)" "$(BENEFIT_REGION)" "$(FILEPATH)"
+	$(PY_RUN_CMD) ingest-edd-web "$(DATASET_ID)" "$(BENEFIT_PROGRAM)" "$(BENEFIT_REGION)" "$(FILEPATH)" $(INGEST_ARGS)
```

Comment:
does `$(INGEST_ARGS)` also need to be in quotes?

Reply:
No, it is intentionally not in quotes in case there are multiple arguments.